LibreNMS/Util/CiHelper.php

```diff
@@ -273,6 +274,12 @@ public function checkLint()
             $php_lint_cmd = array_merge($php_lint_cmd, $files);
 
             $return += $this->execute('PHP lint', $php_lint_cmd);
+
+            if (! $this->flags['lint_skip_phpstan']) {
+                $phpstan_cmd = [$this->checkPhpExec('phpstan'), 'analyze', '--no-interaction', '--error-format=github',  '--memory-limit=2G'];
```

Comment:
Better output style for humans
```suggestion
                $phpstan_cmd = [$this->checkPhpExec('phpstan'), 'analyze', '--no-interaction',  '--memory-limit=2G'];
```

Reply:
Indeed :)